### DISCUSSION TITLE
Error: Failed to load external module next-auth: Error [ERR_MODULE_NOT_FOUND]:

### DISCUSSION THREAD

**Discussion by wangfulong** [nextauthjs/next-auth#12102] [Help]
<img width="975" alt="image" src="https://github.com/user-attachments/assets/6aafd860-9962-4495-92ba-4c615edde618">

"next": "15.0.1-canary.1",
"next-auth": "5.0.0-beta.24"

<img width="923" alt="image" src="https://github.com/user-attachments/assets/39e2b874-28c5-43df-98bc-7a77f78deceb">

I upgraded Next.js from version 14 to 15, and I followed the documentation to upgrade NextAuth as well. Now I'm stuck at this step

**Comment by k-taro56** [comment]
Try to remove the `node_modules` directory and install dependencies.

**Comment by sKopheK** [comment]
adding 

```
transpilePackages: ['next-auth'],
```

to   `next.config.js` helped me
